.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+
+jobs:
+  code:
```

```suggestion
  lint:
```

Esse nome me parece mais adequado.